deeplabcut/pose_estimation_tensorflow/predict_videos.py

```diff
@@ -1808,16 +1808,21 @@ def convert_detections2tracklets(
                     window_size=window_size,
                     identity_only=identity_only,
                 )
-                if calibrate:
-                    trainingsetfolder = auxiliaryfunctions.get_training_set_folder(cfg)
-                    train_data_file = os.path.join(
-                        cfg["project_path"],
-                        str(trainingsetfolder),
-                        "CollectedData_" + cfg["scorer"] + ".h5",
-                    )
-                    ass.calibrate(train_data_file)
-                ass.assemble()
-                ass.to_pickle(dataname.split(".h5")[0] + "_assemblies.pickle")
+                if not os.path.exists(dataname.split(".h5")[0] + "_assemblies.pickle") or overwrite:
+                    if calibrate:
+                        trainingsetfolder = auxiliaryfunctions.get_training_set_folder(cfg)
+                        train_data_file = os.path.join(
+                            cfg["project_path"],
+                            str(trainingsetfolder),
+                            "CollectedData_" + cfg["scorer"] + ".h5",
+                        )
+                        ass.calibrate(train_data_file)
+                    ass.assemble()
+                    ass.to_pickle(dataname.split(".h5")[0] + "_assemblies.pickle")
```

Comment:
Since the file path is repeated three times, I'd define above before the `if`, and use it where needed.

Reply:
You could move `ass_filename` up actually, and reuse this variable.

Reply:
`ass_` is not a great variable name, please change throughout ;)